tracer/src/Datadog.Trace/Ci/Processors/TestSuiteVisibilityProcessor.cs

```diff
@@ -43,10 +45,10 @@ public ArraySegment<Span> Process(ArraySegment<Span> trace)
             }
         }
 
-        return spans is null ? new ArraySegment<Span>(Array.Empty<Span>()) : new ArraySegment<Span>(spans, 0, spIdx);
+        return spans is null ? new ArraySegment<Span>([]) : new ArraySegment<Span>(spans, 0, spIdx);
     }
 
-    public Span Process(Span span)
+    public Span? Process(Span span)
```

Comment:
Should this take a `Span?` given the first check?

Reply:
done in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)